README.md
# Testing t3kit with composer
### t3kit with composer.json and without git submodules.

### Getting Started:

- `git clone https://github.com/t3kit/t3kit_composer.git`
- `cd t3kit_composer`

There is two options for installing composer.

1. I want to have the latest stable release from TYPO3:s extension repository:
- `composer install --no-dev`

2. The second option is I want to develop and contribute then use the following install command:
- `composer install`

### Or if you want to install t3kit with predefined database using [Vagrant](https://github.com/t3kit/t3kit_vagrant):

- `git clone https://github.com/t3kit/t3kit_vagrant.git`
- `cd t3kit_vagrant`
- `./initWithComposer.sh`
- `vagrant up`
- open in browser: `localhost:8081`


CHANGELOG.md

#### v0.2.2 `April 11, 2017`
- **[TASK]** use theme_t3kit v1.10.1 instead of master branch (t3kit 7) ([ba000a9](https://github.com/t3kit/t3kit_composer/commit/ba000a9))
- **[BUGFIX]** Update news to secure version 5.3.3 (Add caseinsensitive security check for overwriteDemandObject) ([3d08311](https://github.com/t3kit/t3kit_composer/commit/3d08311))

***

#### v0.2.1 `February 21, 2017`
- **[FIX]** set correct dependencies for t3kit-extension-tools fix #22 ([9cbdcdd](https://github.com/t3kit/t3kit_composer/commit/9cbdcdd))

***

#### v0.2.0 `February 20, 2017`
- **[TASK]** Change origin for pxa_newsletter_subscription to be Github (#20) ([c8989f5](https://github.com/t3kit/t3kit_composer/commit/c8989f5))
- **[TASK]** Change origin for pxa_newsletter_subscription to be Github ([a75fe9e](https://github.com/t3kit/t3kit_composer/commit/a75fe9e))
- **[UPDATE]** upgrade News from 4.3.0 to 5.3.2 (#18) ([77852a0](https://github.com/t3kit/t3kit_composer/commit/77852a0))
- **[UPDATE]** realurl to v2.0.15 ([34e34ba](https://github.com/t3kit/t3kit_composer/commit/34e34ba))
- **[FEATURE]** Add PackageStage and LocalConfiguration for guide ext (#17) ([077bd0a](https://github.com/t3kit/t3kit_composer/commit/077bd0a))
- **[BUGFIX]** Remove base typo3_console script for composer mode (#16) ([fe8009f](https://github.com/t3kit/t3kit_composer/commit/fe8009f))
- **[BUGFIX]** Fix missing array end block in PackageStates (#15) ([61a27f3](https://github.com/t3kit/t3kit_composer/commit/61a27f3))
- **[FEATURE]** Add guide ext for first configuration tour in theme module (#14) ([6ac762c](https://github.com/t3kit/t3kit_composer/commit/6ac762c))
- **[FEATURE]** Solution for editors to add redirects (#11) ([88c9d9c](https://github.com/t3kit/t3kit_composer/commit/88c9d9c))
- **[BUGFIX]** Change typo3_console composer vendor (#13) ([2bedfd9](https://github.com/t3kit/t3kit_composer/commit/2bedfd9))
- **[FEATURE]** Add typo3-console extension to T3kit (podio feature_55) (#12) ([906d5fb](https://github.com/t3kit/t3kit_composer/commit/906d5fb))
- **[TASK]** Update composer dependency to allow PHP 7.1 ([f55ea4a](https://github.com/t3kit/t3kit_composer/commit/f55ea4a))
- **[UPDATE]** composer dependencies for t3kit 7.10.0 ([6c65ee6](https://github.com/t3kit/t3kit_composer/commit/6c65ee6))

***


typo3conf/PackageStates.php
<?php
# PackageStates.php

# This file is maintained by TYPO3's package management. Although you can edit it
# manually, you should rather use the extension manager for maintaining packages.
# This file will be regenerated automatically if it doesn't exist. Deleting this file
# should, however, never become necessary if you use the package commands.

return [
    'packages' => [
        'core' => [
            'composerName' => 'typo3/cms-core',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/core/',
            'suggestions' => [],
        ],
        'extbase' => [
            'composerName' => 'typo3/cms-extbase',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/extbase/',
            'suggestions' => [],
        ],
        'fluid' => [
            'composerName' => 'typo3/cms-fluid',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/fluid/',
            'suggestions' => [],
        ],
        'documentation' => [
            'composerName' => 'typo3/cms-documentation',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/documentation/',
            'suggestions' => [],
        ],
        'info' => [
            'composerName' => 'typo3/cms-info',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/info/',
            'suggestions' => [],
        ],
        'info_pagetsconfig' => [
            'composerName' => 'typo3/cms-info-pagetsconfig',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/info_pagetsconfig/',
            'suggestions' => [],
        ],
        'extensionmanager' => [
            'composerName' => 'typo3/cms-extensionmanager',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/extensionmanager/',
            'suggestions' => [],
        ],
        'lang' => [
            'composerName' => 'typo3/cms-lang',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/lang/',
            'suggestions' => [],
        ],
        'setup' => [
            'composerName' => 'typo3/cms-setup',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/setup/',
            'suggestions' => [],
        ],
        'rtehtmlarea' => [
            'composerName' => 'typo3/cms-rtehtmlarea',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/rtehtmlarea/',
            'suggestions' => [
                'setup',
            ],
        ],
        'rsaauth' => [
            'composerName' => 'typo3/cms-rsaauth',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/rsaauth/',
            'suggestions' => [],
        ],
        'saltedpasswords' => [
            'composerName' => 'typo3/cms-saltedpasswords',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/saltedpasswords/',
            'suggestions' => [
                'rsaauth',
            ],
        ],
        'func' => [
            'composerName' => 'typo3/cms-func',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/func/',
            'suggestions' => [],
        ],
        'wizard_crpages' => [
            'composerName' => 'typo3/cms-wizard-crpages',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/wizard_crpages/',
            'suggestions' => [],
        ],
        'wizard_sortpages' => [
            'composerName' => 'typo3/cms-wizard-sortpages',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/wizard_sortpages/',
            'suggestions' => [],
        ],
        'about' => [
            'composerName' => 'typo3/cms-about',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/about/',
            'suggestions' => [],
        ],
        'aboutmodules' => [
            'composerName' => 'typo3/cms-aboutmodules',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/aboutmodules/',
            'suggestions' => [],
        ],
        'backend' => [
            'composerName' => 'typo3/cms-backend',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/backend/',
            'suggestions' => [],
        ],
        'belog' => [
            'composerName' => 'typo3/cms-belog',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/belog/',
            'suggestions' => [],
        ],
        'beuser' => [
            'composerName' => 'typo3/cms-beuser',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/beuser/',
            'suggestions' => [],
        ],
        'context_help' => [
            'composerName' => 'typo3/cms-context-help',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/context_help/',
            'suggestions' => [],
        ],
        'cshmanual' => [
            'composerName' => 'typo3/cms-cshmanual',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/cshmanual/',
            'suggestions' => [],
        ],
        'felogin' => [
            'composerName' => 'typo3/cms-felogin',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/felogin/',
            'suggestions' => [],
        ],
        'filelist' => [
            'composerName' => 'typo3/cms-filelist',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/filelist/',
            'suggestions' => [],
        ],
        'filemetadata' => [
            'composerName' => 'typo3/cms-filemetadata',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/filemetadata/',
            'suggestions' => [],
        ],
        'fluid_styled_content' => [
            'composerName' => 'typo3/cms-fluid-styled-content',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/fluid_styled_content/',
            'suggestions' => [],
        ],
        'form' => [
            'composerName' => 'typo3/cms-form',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/form/',
            'suggestions' => [],
        ],
        'frontend' => [
            'composerName' => 'typo3/cms-frontend',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/frontend/',
            'suggestions' => [],
        ],
        'impexp' => [
            'composerName' => 'typo3/cms-impexp',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/impexp/',
            'suggestions' => [],
        ],
        'install' => [
            'composerName' => 'typo3/cms-install',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/install/',
            'suggestions' => [],
        ],
        'lowlevel' => [
            'composerName' => 'typo3/cms-lowlevel',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/lowlevel/',
            'suggestions' => [],
        ],
        'recordlist' => [
            'composerName' => 'typo3/cms-recordlist',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/recordlist/',
            'suggestions' => [],
        ],
        'reports' => [
            'composerName' => 'typo3/cms-reports',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/reports/',
            'suggestions' => [],
        ],
        'scheduler' => [
            'composerName' => 'typo3/cms-scheduler',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/scheduler/',
            'suggestions' => [],
        ],
        'sv' => [
            'composerName' => 'typo3/cms-sv',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/sv/',
            'suggestions' => [],
        ],
        'sys_note' => [
            'composerName' => 'typo3/cms-sys-note',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/sys_note/',
            'suggestions' => [],
        ],
        't3editor' => [
            'composerName' => 'typo3/cms-t3editor',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/t3editor/',
            'suggestions' => [],
        ],
        't3skin' => [
            'composerName' => 'typo3/cms-t3skin',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/t3skin/',
            'suggestions' => [],
        ],
        'tstemplate' => [
            'composerName' => 'typo3/cms-tstemplate',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/tstemplate/',
            'suggestions' => [],
        ],
        'viewpage' => [
            'composerName' => 'typo3/cms-viewpage',
            'state' => 'active',
            'packagePath' => 'typo3/sysext/viewpage/',
            'suggestions' => [],
        ],
        'dyncss' => [
            'composerName' => 'kaystrobach/dyncss',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/dyncss/',
            'suggestions' => [],
        ],
        'dyncss_less' => [
            'composerName' => 'dyncss_less',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/dyncss_less/',
            'suggestions' => [],
        ],
        'static_info_tables' => [
            'composerName' => 'sjbr/static-info-tables',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/static_info_tables/',
            'suggestions' => [],
        ],
        'themes' => [
            'composerName' => 'typo3-themes/themes',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/themes/',
            'suggestions' => [],
        ],
        'gridelements' => [
            'composerName' => 'GridElementsTeam/Gridelements',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/gridelements/',
            'suggestions' => [],
        ],
        't3kit_extension_tools' => [
            'composerName' => 't3kit/t3kit-extension-tools',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/t3kit_extension_tools/',
            'suggestions' => [],
        ],
        'theme_t3kit' => [
            'composerName' => 't3kit/theme-t3kit',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/theme_t3kit/',
            'suggestions' => [],
        ],
        'theme_t3kit_bluemountain' => [
            'composerName' => 't3kit/theme-t3kit-bluemountain',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/theme_t3kit_bluemountain/',
            'suggestions' => [],
        ],
        'realurl' => [
            'composerName' => 'dmitryd/typo3-realurl',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/realurl/',
            'suggestions' => [],
        ],
        'realurl_404_multilingual' => [
            'composerName' => 'realurl_404_multilingual',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/realurl_404_multilingual/',
            'suggestions' => [],
        ],
        'go_maps_ext' => [
            'composerName' => 'clickstorm/go_maps_ext',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/go_maps_ext/',
            'suggestions' => [],
        ],
        'guide' => [
            'composerName' => 'typo3-ter/guide',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/guide/',
            'suggestions' => [],
        ],
        'news' => [
            'composerName' => 'georgringer/news',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/news/',
            'suggestions' => [],
        ],
        'pxa_form_enhancement' => [
            'composerName' => 'pixelant/pxa-form-enhancement',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/pxa_form_enhancement/',
            'suggestions' => [],
        ],
        'pxa_newsletter_subscription' => [
            'composerName' => 'pixelant/pxa-newsletter-subscription',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/pxa_newsletter_subscription/',
            'suggestions' => [],
        ],
        'seo_basics' => [
            'composerName' => 'b13/seo_basics',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/seo_basics/',
            'suggestions' => [],
        ],
        'solr' => [
            'composerName' => 'apache-solr-for-typo3/solr',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/solr/',
            'suggestions' => [],
        ],
        'typo3_console' => [
            'composerName' => 'typo3_console',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/typo3_console/',
            'suggestions' => [],
        ],
        'url_forwarding' => [
            'composerName' => 'patrickbroens/url-forwarding',
            'state' => 'active',
            'packagePath' => 'typo3conf/ext/url_forwarding/',
            'suggestions' => [],
        ],
        'adodb' => [
            'composerName' => 'typo3/cms-adodb',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/adodb/',
            'suggestions' => [],
        ],
        'css_styled_content' => [
            'composerName' => 'typo3/cms-css-styled-content',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/css_styled_content/',
            'suggestions' => [],
        ],
        'dbal' => [
            'composerName' => 'typo3/cms-dbal',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/dbal/',
            'suggestions' => [],
        ],
        'feedit' => [
            'composerName' => 'typo3/cms-feedit',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/feedit/',
            'suggestions' => [],
        ],
        'indexed_search' => [
            'composerName' => 'typo3/cms-indexed-search',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/indexed_search/',
            'suggestions' => [],
        ],
        'indexed_search_mysql' => [
            'composerName' => 'typo3/cms-indexed-search-mysql',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/indexed_search_mysql/',
            'suggestions' => [],
        ],
        'linkvalidator' => [
            'composerName' => 'typo3/cms-linkvalidator',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/linkvalidator/',
            'suggestions' => [],
        ],
        'opendocs' => [
            'composerName' => 'typo3/cms-opendocs',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/opendocs/',
            'suggestions' => [],
        ],
        'recycler' => [
            'composerName' => 'typo3/cms-recycler',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/recycler/',
            'suggestions' => [],
        ],
        'sys_action' => [
            'composerName' => 'typo3/cms-sys-action',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/sys_action/',
            'suggestions' => [],
        ],
        'taskcenter' => [
            'composerName' => 'typo3/cms-taskcenter',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/taskcenter/',
            'suggestions' => [
                'sys_action',
            ],
        ],
        'version' => [
            'composerName' => 'typo3/cms-version',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/version/',
            'suggestions' => [],
        ],
        'workspaces' => [
            'composerName' => 'typo3/cms-workspaces',
            'state' => 'inactive',
            'packagePath' => 'typo3/sysext/workspaces/',
            'suggestions' => [],
        ],
    ],
    'version' => 4,
];


typo3conf/LocalConfiguration.php
<?php
return [
    'BE' => [
        'debug' => false,
        'explicitADmode' => 'explicitAllow',
        'installToolPassword' => '$P$CoQoJHDiVRLGjuMQpOeAdPa6Vwag4o0',
        'loginSecurityLevel' => 'rsa',
        'versionNumberInFilename' => '0',
    ],
    'DB' => [
        'database' => 't3kit',
        'host' => 'localhost',
        'password' => 't3kit1234',
        'socket' => '',
        'username' => 't3kit',
    ],
    'EXT' => [
        'extConf' => [
            'dyncss' => 'a:2:{s:5:"state";s:0:"";s:15:"enableDebugMode";s:0:"";}',
            'dyncss_less' => 'a:0:{}',
            'extension_tools' => 'a:0:{}',
            'filemetadata' => 'a:0:{}',
            'go_maps_ext' => 'a:3:{s:15:"include_library";s:1:"0";s:16:"include_manually";s:1:"0";s:8:"footerJS";s:1:"1";}',
            'gridelements' => 'a:2:{s:20:"additionalStylesheet";s:0:"";s:19:"nestingInListModule";s:1:"0";}',
            'guide' => 'a:0:{}',
            'news' => 'a:17:{s:29:"removeListActionFromFlexforms";s:1:"2";s:20:"pageModuleFieldsNews";s:317:"LLL:EXT:news/Resources/Private/Language/locallang_be.xlf:pagemodule_simple=title,datetime;LLL:EXT:news/Resources/Private/Language/locallang_be.xlf:pagemodule_advanced=title,datetime,teaser,categories;LLL:EXT:news/Resources/Private/Language/locallang_be.xlf:pagemodule_complex=title,datetime,teaser,categories,archive;";s:24:"pageModuleFieldsCategory";s:17:"title,description";s:11:"archiveDate";s:4:"date";s:13:"prependAtCopy";s:1:"1";s:6:"tagPid";s:1:"1";s:25:"showMediaDescriptionField";s:1:"0";s:12:"rteForTeaser";s:1:"0";s:22:"contentElementRelation";s:1:"0";s:13:"manualSorting";s:1:"0";s:19:"categoryRestriction";s:0:"";s:34:"categoryBeGroupTceFormsRestriction";s:1:"0";s:6:"useFal";s:1:"1";s:12:"showImporter";s:1:"0";s:18:"storageUidImporter";s:1:"1";s:22:"resourceFolderImporter";s:12:"/news_import";s:24:"showAdministrationModule";s:1:"1";}',
            'pxa_form_enhancement' => 'a:0:{}',
            'pxa_newsletter_subscription' => 'a:0:{}',
            'realurl' => 'a:5:{s:10:"configFile";s:26:"typo3conf/realurl_conf.php";s:14:"enableAutoConf";s:1:"0";s:14:"autoConfFormat";s:1:"0";s:12:"enableDevLog";s:1:"0";s:19:"enableChashUrlDebug";s:1:"0";}',
            'realurl_404_multilingual' => 'a:0:{}',
            'rsaauth' => 'a:1:{s:18:"temporaryDirectory";s:0:"";}',
            'saltedpasswords' => 'a:2:{s:3:"BE.";a:4:{s:21:"saltedPWHashingMethod";s:41:"TYPO3\\CMS\\Saltedpasswords\\Salt\\PhpassSalt";s:11:"forceSalted";i:0;s:15:"onlyAuthService";i:0;s:12:"updatePasswd";i:1;}s:3:"FE.";a:5:{s:7:"enabled";i:1;s:21:"saltedPWHashingMethod";s:41:"TYPO3\\CMS\\Saltedpasswords\\Salt\\PhpassSalt";s:11:"forceSalted";i:0;s:15:"onlyAuthService";i:0;s:12:"updatePasswd";i:1;}}',
            'scheduler' => 'a:4:{s:11:"maxLifetime";s:4:"1440";s:11:"enableBELog";s:1:"1";s:15:"showSampleTasks";s:1:"1";s:11:"useAtdaemon";s:1:"0";}',
            'seo_basics' => 'a:1:{s:10:"xmlSitemap";s:1:"1";}',
            'solr' => 'a:0:{}',
            'static_info_tables' => 'a:2:{s:13:"enableManager";s:1:"0";s:5:"dummy";s:1:"0";}',
            'theme_core' => 'a:0:{}',
            'theme_t3kit' => 'a:0:{}',
            'theme_t3kit_bluemountain' => 'a:0:{}',
            'themes' => 'a:3:{s:17:"themesIndependent";s:1:"1";s:16:"categoriesToShow";s:230:"theme,languages,colors,container,pages,footer,header,menu,meta,font,socialmedia,forms,extension,bootstrap,theme,siteConstants,site colors,site layout,colors,languages,font,meta,metaDefaults,pages,container,menu,header,footer,forms";s:15:"constantsToHide";s:36:"dateFormat,timeFormat,dateTimeFormat";}',
            'yaml_parser' => 'a:0:{}',
        ],
    ],
    'EXTCONF' => [
        'lang' => [
            'availableLanguages' => [
                'sv',
            ],
        ],
    ],
    'FE' => [
        'debug' => false,
        'lifetime' => '2592000',
        'loginSecurityLevel' => 'rsa',
    ],
    'GFX' => [
        'colorspace' => 'sRGB',
        'im' => 1,
        'im_mask_temp_ext_gif' => 1,
        'im_path' => '/usr/bin/',
        'im_path_lzw' => '/usr/bin/',
        'im_v5effects' => 1,
        'im_version_5' => 'im6',
        'image_processing' => 1,
        'jpg_quality' => '80',
    ],
    'INSTALL' => [
        'wizardDone' => [
            'TYPO3\CMS\Install\Updates\BackendUserStartModuleUpdate' => 1,
            'TYPO3\CMS\Install\Updates\Compatibility6ExtractionUpdate' => 1,
            'TYPO3\CMS\Install\Updates\FileListIsStartModuleUpdate' => 1,
            'TYPO3\CMS\Install\Updates\FilesReplacePermissionUpdate' => 1,
            'TYPO3\CMS\Install\Updates\LanguageIsoCodeUpdate' => 1,
            'TYPO3\CMS\Install\Updates\MigrateMediaToAssetsForTextMediaCe' => 1,
            'TYPO3\CMS\Install\Updates\MigrateShortcutUrlsAgainUpdate' => 1,
            'TYPO3\CMS\Install\Updates\PageShortcutParentUpdate' => 1,
            'TYPO3\CMS\Install\Updates\ProcessedFileChecksumUpdate' => 1,
            'TYPO3\CMS\Install\Updates\TableFlexFormToTtContentFieldsUpdate' => 1,
            'TYPO3\CMS\Rtehtmlarea\Hook\Install\DeprecatedRteProperties' => 1,
            'TYPO3\CMS\Rtehtmlarea\Hook\Install\RteAcronymButtonRenamedToAbbreviation' => 1,
        ],
    ],
    'MAIL' => [
        'transport_sendmail_command' => '/usr/bin/env catchmail -f test@local.dev',
    ],
    'SYS' => [
        'UTF8filesystem' => '1',
        'caching' => [
            'cacheConfigurations' => [
                'extbase_object' => [
                    'backend' => 'TYPO3\\CMS\\Core\\Cache\\Backend\\Typo3DatabaseBackend',
                    'frontend' => 'TYPO3\\CMS\\Core\\Cache\\Frontend\\VariableFrontend',
                    'groups' => [
                        'system',
                    ],
                    'options' => [
                        'defaultLifetime' => 0,
                    ],
                ],
            ],
        ],
        'clearCacheSystem' => false,
        'devIPmask' => '',
        'displayErrors' => 0,
        'enableDeprecationLog' => false,
        'encryptionKey' => '710ba8e4edb2782cda0df7721fac87fd326b96300a24e6899c50e66021eb63bcfb344ace69cca30a4a2096ecdc0bd395',
        'exceptionalErrors' => 28674,
        'isInitialDatabaseImportDone' => true,
        'isInitialInstallationInProgress' => false,
        'sitename' => 'TYPO3 t3kit',
        'sqlDebug' => 0,
        'systemLocale' => 'en_US.utf8',
        'systemLogLevel' => 2,
        't3lib_cs_convMethod' => 'mbstring',
        't3lib_cs_utils' => 'mbstring',
        'textfile_ext' => 'txt,ts,typoscript,html,htm,css,tmpl,js,sql,xml,csv,xlf,less',
    ],
];
